creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -16,10 +16,7 @@ const healthCheck = async ({ libs } = {}, logger, sequelize) => {
     'git': process.env.GIT_SHA,
     'selectedDiscoveryProvider': 'none',
     'creatorNodeEndpoint': config.get('creatorNodeEndpoint'),
-    'spID': config.get('spID'),
-    country: config.get('serviceCountry'),
```

Comment:
don't we need these still? since we're switching to health check?

Reply:
oh right 🧠 💩